features/git-extract/branch_exists_locally.feature

```diff
@@ -12,7 +12,7 @@ Feature: git extract: errors when the branch exists locally
 
   Scenario: with open changes
     Given I have an uncommitted file with name: "uncommitted" and content: "stuff"
-    When I run `git extract existing-feature` while allowing errors
+    When I run `git extract existing-feature`, it errors
     Then it runs the Git commands
```

Comment:
This reads weird now: "When I run `xxx` it errors, then it runs the Git commands..."